src/settings-ui/Settings.UI/SettingsXAML/Views/CmdNotFoundPage.xaml

```diff
@@ -26,86 +26,127 @@
-                <custom:SettingsGroup x:Uid="CmdNotFound_InstallationGroup" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
-                    <controls:SettingsCard
-                        x:Uid="CmdNotFound_Enable"
-                        HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/FluentIcons/FluentIconsCmdNotFound.png}"
-                        IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
-                        <StackPanel Orientation="Horizontal" Spacing="8">
-                            <SymbolIcon
-                                Foreground="{ThemeResource SystemFillColorSuccessBrush}"
-                                Symbol="Accept"
-                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToVisibilityConverter}}" />
-                            <SymbolIcon
-                                Foreground="{ThemeResource SystemFillColorCriticalBrush}"
-                                Symbol="Cancel"
-                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+
+                <controls:SettingsExpander
+                    x:Uid="CmdNotFound_Enable"
+                    HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/FluentIcons/FluentIconsCmdNotFound.png}"
+                    IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}"
+                    IsExpanded="True">
+                    <controls:SwitchPresenter TargetType="x:Boolean" Value="{x:Bind ViewModel.IsCommandNotFoundModuleInstalled, Mode=OneWay}">
+                        <controls:Case Value="True">
+                            <StackPanel Orientation="Horizontal" Spacing="8">
+                                <TextBlock
+                                    x:Uid="InstalledLabel"
+                                    VerticalAlignment="Center"
+                                    Foreground="{ThemeResource TextFillColorSecondaryBrush}" />
+                                <FontIcon
+                                    Margin="0,0,4,0"
+                                    VerticalAlignment="Center"
+                                    AutomationProperties.AccessibilityView="Raw"
+                                    Foreground="{ThemeResource SystemFillColorSuccessBrush}"
+                                    Glyph="&#xEC61;" />
+                                <HyperlinkButton x:Uid="CmdNotFound_UninstallButton" Command="{x:Bind ViewModel.UninstallModuleEventHandler}" />
+                            </StackPanel>
+                        </controls:Case>
+                        <controls:Case Value="False">
                             <Button
```

Comment:
@jaimecbernardo One thing to add here is that `IsEnabled` need to be set to false whenever one of the two modules are not detected?